client/js/lounge.js

```diff
@@ -253,8 +254,9 @@ $(function() {
 			const args = text.substr(1).split(" ");
 			const cmd = args.shift().toLowerCase();
 
-			if (typeof utils.inputCommands[cmd] === "function" && utils.inputCommands[cmd](args)) {
-				return false;
+			if (commands[cmd]) {
+				commands[cmd].input(...args);
+				return;
```

Comment:
Keep the `return false`. Didn't notice last time.

Reply:
done